### DISCUSSION TITLE
Welcome to game_of_life Discussions!

### DISCUSSION THREAD

**Discussion by Tech-Matt** [Tech-Matt/game_of_life#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PhysicsMD** [comment]
I will say I think there is a slight issue with the algorithm based on the output, but fine so far.
